pyaedt/application/MessageManager.py

```diff
@@ -208,31 +235,33 @@ def add_info_message(self, message_text, level=None):
     @aedt_exception_handler
     def add_debug_message(self, message_text, level=None):
         """
-        .. deprecated use add_info_message
+        .. Deprecated. Use the method `add_info_message`.
```

Comment:
```suggestion
        .. deprecated:: 0.2.0
           Use the method :func:`MessageList.add_info_message`.
```
Here we use two sphinx directives.  The first is the `.. depreciated` directive (see [directive-deprecated](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated)).  It shows up well in the docs.  Secondly, we provide a link to the actual method with the `:func:` directive.  Note how we have to provide the class and then the method name when providing this link.